pkg/daemon/ceph/config/config.go

```diff
@@ -201,8 +201,8 @@ func CreateDefaultCephConfig(context *clusterd.Context, cluster *ClusterInfo, ru
 			RunDir:                 runDir,
 			MonMembers:             strings.Join(monMembers, " "),
 			MonHost:                strings.Join(monHosts, ","),
-			LogFile:                "/dev/stdout",
-			MonClusterLogFile:      "/dev/stdout",
+			LogFile:                "/dev/stderr",
```

Comment:
this means that all the ceph logging will be written to stderr instead of stdout? sounds like this is what we've been missing to run with debug logging on. Is the rgw config the only place that was failing with debug logging enabled? I wondered if there were other places too.

Reply:
it certainly would cause problems whereever a Rook construct wanted to try to capture stdout